src/app/resourcebundles/data/consumption/hi.properties

```diff
@@ -1990,4 +1990,6 @@ frmelmnts.lbl.of = का
 frmelmnts.lbl.boardsTest=Board {frameworkCategory1}
 frmelmnts.lbl.mediumTest=Medium {frameworkCategory2}
 frmelmnts.lbl.gradeTest=Grade {frameworkCategory3}
-frmelmnts.lbl.boardsTestWithText=बोर्ड {frameworkCategory3}
\ No newline at end of file
+frmelmnts.lbl.boardsTestWithText=बोर्ड {frameworkCategory3}
```

Comment:
with out the JSON file the properties file will not work

Reply:
we only added the last line `frmelmnts.lbl.questionSets = प्रश्न सेट`. Above mentioned line is already there

Reply:
@rajeevsathish can you please merge this PR. As Manoj mentioned we have not added the line on which you have shared the comment. 
Thanks